### PR TITLE
Remove unused AddonForm class

### PR DESCRIPTION
**Pull request by diox**
This form class appears to be unused and is confusing my greps through the code :)

### CODE REVIEW

**Comment by muffinresearch** [comment]
r+

**Comment by EnTeQuAk** [comment]
lgtm

File: src/olympia/addons/forms.py [hunk 1]
```diff
@@ -397,40 +396,6 @@ class Meta:
...
 
 
-class AddonForm(happyforms.ModelForm):
-    name = forms.CharField(widget=TranslationTextInput,)
-    homepage = forms.CharField(widget=TranslationTextInput, required=False)
-    eula = forms.CharField(widget=TranslationTextInput,)
-    description = forms.CharField(widget=TranslationTextInput,)
-    developer_comments = forms.CharField(widget=TranslationTextInput,)
-    privacy_policy = forms.CharField(widget=TranslationTextInput,)
-    the_future = forms.CharField(widget=TranslationTextInput,)
```
Comment (andymckay) [inline src/olympia/addons/forms.py]: I'm sad that we have no future anymore.

Comment (diox) [inline src/olympia/addons/forms.py]: Don't worry, there are 106 more occurrences of this field in the repos, the future is not going anywhere :)

**Pull request merged**